.appveyor.yml

```diff
@@ -4,7 +4,7 @@ platform: x64
 
 environment:
     matrix:
-        - QTDIR: C:\Qt\5.9\msvc2017_64
+        - QTDIR: C:\Qt\5.12.2\msvc2017_64
```

Comment:
I still would like to support older version of Qt, as old as possible.

Reply:
I would like it as well, but qmake only supports C++17 since Qt 5.12.x.
As for now this branch requires C++17 all older versions are out of option unless we use Qbs for that.